### ISSUE TITLE
Unknown option "config" with value "undefined" was found.

### ISSUE THREAD

**Issue by stepankuzmin** [prettier/prettier-eslint-cli#125]
- `prettier-eslint-cli` version: 8.8.1
- `prettier` version: 1.10.2
- `eslint` version: 4.17.0

What you did:

```shell
prettier-eslint --write "src/**/*.js"
```

What happened:

```shell
Validation Warning:

  Unknown option "config" with value "undefined" was found.
  This is probably a typing mistake. Fixing it will remove this message.
```

**P.S.** It outputs this warning for every file. It seems that `prettier-eslint-cli` sends unnecessary option `config`  in `prettierOptions`.

**Comment by sQVe** [comment]
This is already covered by #123.

**Comment by stepankuzmin** [comment]
This is not the same because this warning is about prettier options, not the rules.

**Comment by zimme** [comment]
I'll accept a PR for this, otherwise I hope I have time to fix this over the weekend.

**Comment by ReDrUm** [comment]
Does this lib wish to support the [prettier --config path](https://prettier.io/docs/en/cli.html#find-config-path-and-config)? It's provided as a property but it's not yet utilised which is causing this issue.

We can either remove it entirely by deleting the `config` property within [`parser.js`](https://github.com/prettier/prettier-eslint-cli/blob/master/src/parser.js#L74)

Or we can declare it so that it's not included in the Object spread for`...prettierOptions` inside [`format-files.js`](https://github.com/prettier/prettier-eslint-cli/blob/master/src/format-files.js#L57) (of course then we should wire it up too). e.g.

``` javascript
 prettierLast,
config: prettierConfigPath,`
  ...prettierOptions
```

I'll whip up a PR for the removal of the unused property in the meantime.

**Comment by fzaninotto** [comment]
Si it seems that 63f621f188f023a0f8decdf8bf0223c3a2698f32 fixes this issue, but #128 is a similar bug and causes too many warnings
